ncwriter/schema.py

```diff
@@ -26,77 +28,27 @@
                                       default_types=types
                                       )
 
+TEMPLATE_SCHEMA_JSON = os.path.join(os.path.dirname(__file__), 'template_schema.json')
```

Comment:
Maybe look at the package_data mechanism for including non-Python files?

Reply:
Yep, I'm looking into it.

In terms of accessing the including file using setuptoos functions, [this](https://setuptools.readthedocs.io/en/latest/pkg_resources.html?highlight=data#overview) seems to suggest they rely on having "egg-info" metadata directories. Will that be a problem, given that we're using a wheel to install the package?